components/atom/imagePanel/src/index.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const BASE_16 = 16
+const HORIZONTAL_ALIGNMENTS = {
+  LEFT: 'left',
+  CENTER: 'center',
+  RIGHT: 'right'
+}
+
+const VERTICAL_ALIGNMENTS = {
+  TOP: 'top',
+  CENTER: 'center',
+  BOTTOM: 'bottom'
+}
+
+const TYPES = {
```

Comment:
not used

Reply:
It's being exported and therefore "used"

Reply:
We found it out later LOL